pipeline-runner/src/data-ingest/0-download_gem2s.r

```diff
@@ -60,4 +61,6 @@ task <- function(input, pipeline_config) {
     message(exportJSON)
     write(exportJSON, "/input/meta.json")
     message('Written config json')
+
+    return(list())
```

Comment:
because I was trying to follow roughly the way qc was implemented to bring them closer together, so each task returns data that it wants to be sent over sns to the api.

In most cases it is nothing (an empty list), in the case of the last step it is the dyanmodb experiment.

After this, we append the values that we know we always want to send over sns, the task name and the experiment id